pydss/flopper.py

```diff
@@ -126,10 +126,10 @@ def dent(self, bid_id, lot, bid):
     self.bids[bid_id].tic = int(self.now)+ self.ttl
 
 
-def deal(self, sid):
+def deal(self, sid,lot):
     require(self.live == 1,"Flapper/not-live")
     require(self.bids[sid].tic != 0 and (self.bids[sid].tic < self.now or self.bids[sid].end < self.now), "Flapper/not-finished")
-    self.vat.move(sid, self.bids[sid].guy, self.bids[sid].lot);
+    self.vat.transferFrom(sid, self.bids[sid].guy, lot);
```

Comment:
you can remove this line entirely, in the source code, there's no call to `vat.move`, just the call below to `gem.mint`